tests/tape/test_reversible.py

```diff
@@ -289,37 +289,10 @@ def test_multiple_rx_gradient(self, tol):
         expected_jacobian = -np.diag(np.sin(params))
         assert np.allclose(circuit_jacobian, expected_jacobian, atol=tol, rtol=0)
 
-    qubit_ops = [getattr(qml, name) for name in qml.ops._qubit__ops__]
-    analytic_qubit_ops = {cls for cls in qubit_ops if cls.grad_method == "A"}
-    analytic_qubit_ops = analytic_qubit_ops - {
-        qml.CRX,
-        qml.CRY,
-        qml.CRZ,
-        qml.CRot,
-        qml.PhaseShift,
-        qml.ControlledPhaseShift,
-        qml.CPhase,
-        qml.PauliRot,
-        qml.MultiRZ,
-        qml.U1,
-        qml.U2,
-        qml.U3,
-        qml.IsingXX,
-        qml.IsingYY,
-        qml.IsingZZ,
-        qml.SingleExcitation,
-        qml.SingleExcitationPlus,
-        qml.SingleExcitationMinus,
-        qml.DoubleExcitation,
-        qml.DoubleExcitationPlus,
-        qml.DoubleExcitationMinus,
-        qml.OrbitalRotation,
-    }
```

Comment:
This was funny: the logic above leaves `analytic_qubit_ops` with only 2-3 operations anyways :)

Reply:
lmao 🤣